moztelemetry/dataset.py

```diff
@@ -40,6 +41,30 @@ def _group_by_size_greedy(obj_list, tot_groups):
         current_group.append(obj)
     return groups
 
+def _group_by_equal_size(obj_list, tot_groups, threshold=pow(2, 32)):
+    sub_list = []
+    groups = [[]]
+    pq = []
+    tmp_object = none
+
+    if tot_groups <= 1:
+        group_by_size_greedy(obj_list, tot_groups)
+
+    pq = heapq.heapify(obj_list)
+
+    while pq: 
+        if pq[0].size >= threshold:
+            tmp_object = heapq.heappop(pq)
+            groups.append(tmp_object)
+    else:
+        if sub_list.size >= threshold:
```

Comment:
The `sub_list` doesn't have a `.size` method.

Reply:
Thanks for catching this. I meant to look that up for proper syntax.